clap_complete/tests/testsuite/fish.rs

```diff
@@ -194,26 +194,27 @@ hint                                                               -h  (Print he
     let actual = runtime.complete(input, &term).unwrap();
     assert_data_eq!(actual, expected);
 
-    let input = "exhaustive quote \t";
+    let input = "exhaustive quote \t\t";
     let expected = snapbox::str![
-        r#"% exhaustive quote
-cmd-backslash                                        (Avoid '\n')
+        r#"% exhaustive quote 
+cmd-backslash                                        (Avoid '/n')
```

Comment:
Could you please explain why you want to change `\n` to `/n`? Aren't we typically using `\` for escape sequences?

Reply:
snapbox, by default, turns `\` into `/` to normalize windows/linux path separators